src/CsvHelper/Configuration/CsvConfiguration.cs

```diff
@@ -28,6 +28,8 @@ public record CsvConfiguration : IReaderConfiguration, IWriterConfiguration
 		/// <inheritdoc/>
 		public virtual BadDataFound BadDataFound { get; set; } = ConfigurationFunctions.BadDataFound;
 
+		public virtual bool ProcessBadDataFields { get; set; } = true;
```

Comment:
All the other properties (not only properties) have ```/// <inheritdoc/>``` above them, you forgot to add it above this line